skunkworks_crow/src/main/java/org/odk/share/tasks/DownloadJob.java

```diff
@@ -362,9 +363,18 @@ private String getInstancesPath() {
 
     private String getOdkDestinationDir() {
         SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getContext().getApplicationContext());
+        String directoryPath = null;
 
-        return prefs.getString(PreferenceKeys.KEY_ODK_DESTINATION_DIR,
-                getContext().getString(R.string.default_odk_destination_dir));
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
Did you check this behavior on any of the lower-end devices, like Android 4.4(Kitkat) or JellyBean (4.3) or Android 5.1 (Lollipop)?

Reply:
Yeah I checked in android 5 it is working perfectly with edit text

Reply:
I would suggest you to test on Android 4.4, I think this will fail on that device.

Reply:
Sorry I dont have Android 4.4 device I checked in 5,6,7,8,9 it is working perfectly

Reply:
But I send my branch to onother guy he tested and he said that it is working with edit text perfectly as before